### ISSUE TITLE
What is the proper way to insert line break in expressions?

### ISSUE THREAD

**Issue by qianbinbin** [Keats/tera#625]
I was writing a zola shortcode to highlight codes from file, like this:
```
{% set source = load_data(path="path/to/hello.c") %}
{{ "```c\n" ~ source ~ "\n```" | markdown | safe }}
```
But the `\n` doesn't work, just appears as "\n" rather than line break. What is the proper way to implement this? Thanks!

**Comment by Keats** [comment]
Hm try to put the `| safe` before the `| markdown` as well?

**Comment by acheronfail** [comment]
Note that I've found a workaround: replace `\n` with a literal newline, eg:

```html
<pre>{{ "hello" ~ "
" ~ "world" }}</pre>
```

Not the nicest, but hey it works.

**Comment by Keats** [comment]
Probably too much escaping going on.

**Comment by svallory** [comment]
Just hit the same issue trying to run `- {{ outputs | join(sep='\\n     -') }}`

**Comment by Keats** [comment]
Yeah it's not working in the current version. v2 has it working already

**Comment by Keats** [comment]
Closing as part of the Tera v2 release. If this is still relevant in v2, please reopen — see the 
  migration guide: https://github.com/Keats/tera/blob/master/MIGRATION.md
